tableauserverclient/models/tableau_auth.py

```diff
@@ -1,41 +1,70 @@
-class TableauAuth(object):
+class Credentials:
+    def __init__(self, site_id=None, user_id_to_impersonate=None):
+        self.site_id = site_id or ""
+        self.user_id_to_impersonate = user_id_to_impersonate or None
+
+    @property
+    def credentials(self):
+        credentials = "Credentials can be username/password, Personal Access Token, or JWT"
+        +"This method returns values to set as an attribute on the credentials element of the request"
+
+    def __repr__(self):
+        display = "All Credentials types must have a debug display that does not print secrets"
+
+
+def deprecate_site_attribute():
```

Comment:
At this point, maybe we should just pick a release to kill it as well, it's been many years.

Reply:
Let's call it 0.20, which should come about October.